.goreleaser-template.yaml

```diff
@@ -69,6 +71,7 @@ builds:
       - "6"
       - "7"
     ignore:
+      - goos: linux
```

Comment:
Why do we need to ignore this here?

Reply:
When GOOS=linux, we'll replace `REPLACE_WITH_RELEASE_GOOS` in both `id=builds-linux` and `id=builds-other`; we don't want to build Linux in the other pipeline and similarly don't want to build non-linux in the first pipeline, so we exclude each OS from the others' pipelines.

Reply:
Ah got it